apps/api/src/app/workflows-v2/shared/schemas/email-control.schema.ts

```diff
@@ -28,3 +31,8 @@ export const emailStepUiSchema: UiSchema = {
     skip: skipControl.uiSchema.properties.skip,
   },
 };
+
+export const emailStepControl = {
```

Comment:
I shared the same comment in a previous PR, I don't think we need this extra export. It's adds extra bloat.

Reply:
done! :)